packages/cli/test/services/stub-publisher.test.ts

```diff
@@ -17,7 +17,7 @@ const rewire = require('rewire')
 describe('stub publisher', () => {
```

Comment:
Could you add the new command to the documentation?

Reply:
Hey @gonzalogarciajaubert I would love to, but there is no CLI commands section 😢 
What do you think would be a good place to mention this command?

Reply:
You are right! We have two options:

1. The features section: http://docs.booster.cloud/chapters/04_features
2. The going deeper section: http://docs.booster.cloud/chapters/05_going-deeper: 

I think this is a Booster extension so the `going deeper` is the best option.